rs/backend/src/stats.rs

```diff
@@ -51,6 +51,9 @@ pub struct Stats {
     pub schema: Option<u32>,              // The numeric form of a SchemaLabel.
     pub migration_countdown: Option<u32>, // When non-zero, a migration is in progress.
     pub exceptional_transactions_count: Option<u32>,
+    // TODO: Delete this once the stable memory migration is complete.  This is used purely to get
+    // an idea of how long migration is likely to take.
```

Comment:
"How long" sounds like we're measuring time. But IIUC, we're counting the number of calls. Is that right?

If so, consider adjusting the comment, e.g.:

```suggestion
    // a counter of periodic tasks executed since last upgrade
```

Reply:
I am actually using it to estimate how long, in time, a migration is likely to take. If it's below a week, current parameters are fine.  If it's above we might want to adjust them a bit.

Reply:
How does one go from `periodic_tasks_run` to something time-related?

Reply:
The counter is monitored, so I can use Prometheus to look up how much the counter grows in a given wall clock interval.